module.json

```diff
@@ -1,21 +1,21 @@
 {
   "name": "mbed-client-c",
   "version": "4.0.0",
-  "description": "Nanostack NSDL and COAP library",
+  "description": "Nanostack NSDL library",
   "keywords": [
     "coap",
     "nanostack"
   ],
   "license": "Apache-2.0",
   "extraIncludes": [
     "nsdl-c",
-    "source/libNsdl/src/include",
-    "source/libCoap/src/include"
+    "source/libNsdl/src/include"
   ],
   "dependencies": {
     "nanostack-libservice": "^3.0.0",
     "mbed-trace": ">=0.2.0,<2.0.0",
-    "nanostack-randlib": "^1.2.0"
+    "nanostack-randlib": "^1.2.0",
+    "libcoap": "^1.0.0"
```

Comment:
As per discussion, libcoap versioning should really continue from the same point as nsdl, so 5.0.0.